### PR TITLE
Remove "COMPLETED" Option for Super User Task Creation

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
### Issue: Closes https://github.com/Real-Dev-Squad/website-status/issues/976

**Design doc:** https://docs.google.com/document/d/1C3oizE01vH6cQHRtoJbA7-QHDoDhHuPiMllqvh6v5cM/edit?usp=sharing (Point 1 of status site)

### Description:

- When a super user accesses the task creation form on https://status.realdevsquad.com/issues?dev=true, they should see only the "DONE" option among DONE and COMPLETED in the status selection tag under feature flag.

Backend changes
- [ ] Yes
- [x] No

Frontend Changes
- [x] Yes
- [ ] No

### Dev Tested:
- [x] Yes
- [ ] No

### Is Under Feature Flag 
- [x] Yes
- [ ] No

## Test Stats

image

### Images/video of the change:

Before Change:

https://github.com/Real-Dev-Squad/website-status/assets/107163260/60f64d43-d5db-4715-b2cc-90c486575239

After Change:

https://github.com/Real-Dev-Squad/website-status/assets/107163260/e6e790a9-ad5a-469a-a096-a10225425418

### CODE REVIEW

**Comment by ajoykumardas12** [comment]
Please add test stats too.

> they should see only the "DONE" option in the status selection tag.

This is a bit confusing, they should see only "DONE" among "DoONE" and "COMPLETED" is better I think.

**Comment by shubhamsigdar1** [comment]
> Please add test stats too.
> 
> > they should see only the "DONE" option in the status selection tag.
> 
> This is a bit confusing, they should see only "DONE" among "DoONE" and "COMPLETED" is better I think.

Done with changes

File: src/components/issues/ActionForm.tsx [hunk 1]
```diff
@@ -68,6 +68,11 @@ const ActionForm: FC<ActionFormProps> = ({
     createTask,
     updateTask,
 }) => {
+    const router = useRouter();
+    const devMode = router.query.dev === 'true' ? true : false;
```
[the remark this answers is not in the document]
Comment (shubhamsigdar1) [reply src/components/issues/ActionForm.tsx]: Done with changes can you please check

**Review by shubhamjaintech — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

**Review by ajoykumardas12 — APPROVED** [review]

**Review by iamitprakash — CHANGES_REQUESTED** [review]
PR description are missing like UnderFeatureFalg and all....

**Comment by shubhamsigdar1** [comment]
> PR description are missing like UnderFeatureFalg and all....

Added

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**